charts/kubefed/README.md

```diff
@@ -129,6 +129,10 @@ chart and their default values.
 | controllermanager.certManager.rootCertificate.organizations       | Specifies the list of organizations to include in the cert-manager generated root certificate.                                                                  | []                              |
 | controllermanager.certManager.rootCertificate.commonName     | Specifies the CN value for the cert-manager generated root certificate.                                                                                              | ca.webhook.kubefed              |
 | controllermanager.certManager.rootCertificate.dnsNames       | Specifies the list of subject alternative names for the cert-manager generated root certificate.                                                                     | ["ca.webhook.kubefed"]          |
+| controllermanager.postInstallJob.repository        | Repo of the kubectl post-install job container                                                                                                                                 | bitnami                         |
```

Comment:
it is `image` repo, name and tag instead of `container` I think. This is also more consistent with other similar items in values.
So, I'd like to have 'Repo of the kubectl image for post-install job', etc.

Reply:
Updated the descriptions, thanks!